src/ensembl/src/content/app/browser/Browser.tsx

```diff
@@ -120,26 +117,30 @@ export const Browser = (props: BrowserProps) => {
 
   return (
     <ApolloProvider client={client}>
-      <div className={styles.browserInnerWrapper}>
-        <BrowserAppBar onSpeciesSelect={changeGenomeId} />
-        {props.browserQueryParams.focus ? (
-          <StandardAppLayout
-            mainContent={mainContent}
-            sidebarContent={<TrackPanel />}
-            sidebarNavigation={<TrackPanelTabs />}
-            sidebarToolstripContent={<TrackPanelBar />}
-            onSidebarToggle={onSidebarToggle}
-            topbarContent={<BrowserBar />}
-            isSidebarOpen={props.isTrackPanelOpened}
-            isDrawerOpen={props.isDrawerOpened}
-            drawerContent={<Drawer />}
-            onDrawerClose={toggleDrawer}
-            viewportWidth={props.viewportWidth}
-          />
-        ) : (
-          <ExampleObjectLinks {...props} />
-        )}
-      </div>
+      {!props.activeGenomeId ? (
+        <BrowserInterstitial />
```

Comment:
Some general questions I wanted to address.

First, isn't it true that what we think about as an interstitial is shown in two situations:

1) When the user visiting the relevant "app" has not even selected a single species
2) When the user has selected a species, but has not yet selected a gene or region of interest

In the first case, we show a screen with instructions for how to select a species ([link to XD](https://xd.adobe.com/view/36dd8d77-32a6-48c5-a833-3b34d285bbe3-638e/screen/88b66037-ac26-4419-a590-21754a64201a?fullscreen)); in the second case, we show an in-app search screen ([link to XD](https://xd.adobe.com/view/36dd8d77-32a6-48c5-a833-3b34d285bbe3-638e/screen/c621bab7-7ed3-49f4-afe7-e4a3be06b58e/?fullscreen)). Both are the functions of an interstitial, aren't they?

The code that I am seeing in this PR narrowly specializes the interstitial to handle only case 1. For example, it calls the grey panel that contains the instructions an instructions panel, although the same panel will contain in-app search field in the future. Is this a good idea?

![image](https://user-images.githubusercontent.com/6834224/122040410-ab40b100-cdcf-11eb-8a48-a0bd10a9577b.png)

Second, shouldn't the grey panel in the interstitial be the same height as the grey panel in the species selector?

Third, wouldn't it be a good idea to temporarily move the example links block from the browser component into the new interstitial component, so that it becomes clearer that it handles the two cases? The example links will be removed from the interstitial when we add in-app search into it; but at least having them in the interstitial will remind us of the two scenarios that it needs to support.

Reply:
Yes I like the idea of moving Example links to the interstitial page B (with search). 
Except that the statement  "The example links will be removed from the interstitial when we add in-app search into it"  which is wrong as per our code review discussion with Andrea.